update-readme.js

```diff
@@ -18,7 +18,7 @@ function main() {
     const readme_m = /^([^]+## Options)\n[^]*?\n(##[^#][^]*|\s*)$/.exec(readme_input);
     assert(readme_m);
 
-    const full_help = (child_process.execSync('./run --help', {
+    const full_help = (child_process.execSync('node run --help', {
```

Comment:
If `./run` doesn't work, this points to a much larger problem. Why doesn't `./run` work on macOS?

Reply:
This is the full output:

```bash
m.hagemeister ~/dev/github/pentf (master) $ ./update-readme.js 
env: node: No such file or directory
child_process.js:650
    throw err;
    ^

Error: Command failed: ./run --help
env: node: No such file or directory

    at checkExecSyncError (child_process.js:611:11)
    at Object.execSync (child_process.js:647:15)
    at main (/Users/m.hagemeister/dev/github/pentf/update-readme.js:21:38)
    at Object.<anonymous> (/Users/m.hagemeister/dev/github/pentf/update-readme.js:47:1)
    at Module._compile (internal/modules/cjs/loader.js:1176:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1196:10)
    at Module.load (internal/modules/cjs/loader.js:1040:32)
    at Function.Module._load (internal/modules/cjs/loader.js:929:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  status: 127,
  signal: null,
  output: [
    null,
    <Buffer >,
    <Buffer 65 6e 76 3a 20 6e 6f 64 65 3a 20 4e 6f 20 73 75 63 68 20 66 69 6c 65 20 6f 72 20 64 69 72 65 63 74 6f 72 79 0a>
  ],
  pid: 25779,
  stdout: <Buffer >,
  stderr: <Buffer 65 6e 76 3a 20 6e 6f 64 65 3a 20 4e 6f 20 73 75 63 68 20 66 69 6c 65 20 6f 72 20 64 69 72 65 63 74 6f 72 79 0a>
}
```

Reply:
Seems to be a well-known issue of macOS, https://discuss.atom.io/t/how-to-execute-node-js-child-process-from-package/4880